<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
background-color: rgb(197, 201, 255);
border-bottom: 1px solid;
border-color: rgb(92, 95, 142);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="5" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">QWidget {
background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(238,174,202,1), stop:1 rgba(148,187,233,1));
border: none;
}</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageMenu">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>93</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QWidget" name="widgetMenu" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>300</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget {
background:none;
}

QPushButton {
font: 24pt &quot;Drexs&quot;;
color: black;
border-style: outset;
border-width: 2px;
border-radius: 12px;
border-color: rgba(130, 127, 255, 133);
background-color: rgba(255, 255, 255, 95);
}

QPushButton:pressed {
background-color: rgb(182, 183, 255);
border-style: inset;
}</string>
          </property>
          <widget class="QPushButton" name="pushButtonToCalculator">
           <property name="geometry">
            <rect>
             <x>19</x>
             <y>30</y>
             <width>161</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Calculator</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonToCredit">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>161</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Credit</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonToDeposit">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>161</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Deposit</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonExit">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>230</y>
             <width>161</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Quit</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>93</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>375</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>375</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="Calculator" name="pageCalculator"/>
      <widget class="CreditCalculator" name="pageCredit"/>
      <widget class="QWidget" name="pageDeposit"/>
     </widget>
    </item>
    <item row="2" column="0" rowspan="3">
     <widget class="QWidget" name="widgetTopMenu" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonMenu">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>69</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
font: 12pt &quot;Drexs&quot;;
color: black;
border: none;
background: none;
}

QPushButton:pressed {
background-color: rgb(182, 183, 255);
border-color: rgba(130, 127, 255, 133);
border-style: inset;
border-width: 2px;
border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string>Menu</string>
         </property>
         <property name="icon">
          <iconset resource="../misc/resource.qrc">
           <normaloff>:/icons/menu_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/menu_FILL0_wght400_GRAD0_opsz24.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>17</width>
           <height>17</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionCalculator"/>
    <addaction name="actionCredit"/>
    <addaction name="actionDeposit"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCalculator">
   <property name="text">
    <string>Calculator</string>
   </property>
  </action>
  <action name="actionCredit">
   <property name="text">
    <string>Credit</string>
   </property>
  </action>
  <action name="actionDeposit">
   <property name="text">
    <string>Deposit</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Calculator</class>
   <extends>QWidget</extends>
   <header>calculator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreditCalculator</class>
   <extends>QWidget</extends>
   <header>creditcalculator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../misc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
